skellytracker/trackers/openpose_tracker/openpose_tracker.py

```diff
@@ -132,27 +132,13 @@ def process_video(
 if __name__ == "__main__":
     # Example usage
 
-    input_video_folder = Path(
-        r"C:\Users\aaron\FreeMocap_Data\recording_sessions\freemocap_sample_data"
-    )
-    input_video_filepath = (
-        input_video_folder
-        / "synchronized_videos"
-        / "sesh_2022-09-19_16_16_50_in_class_jsm_synced_Cam1.mp4"
-    )
-
-    output_video_filepath = input_video_folder / "openpose_annotated_videos"
-    output_video_filepath.mkdir(parents=True, exist_ok=True)
 
-    output_json_path = (
-        input_video_folder / "output_data" / "raw_data" / "openpose_jsons"
-    )
+    openpose_root_folder_path = r"C:\openpose"
+    output_json_path = Path(r'C:\openpose\output_json')
     output_json_path.mkdir(parents=True, exist_ok=True)
+    input_video_filepath = r'C:\path\to\input\video.mp4'
+    output_video_filepath = r'C:\path\to\output\video.mp4'
```

Comment:
output file path should be `avi` not `mp4`. Happy to make the change on my end, just lmk

Reply:
fixed this. I had just uncommented the example paths